assets/js/select_seats.js

```diff
@@ -100,7 +100,7 @@ function setupSelectSeats(config) {
                 info = [
                     "<div>", seatObject.name, "<br>",
                     "Student: ", pricing["student"], "kr", "<br>",
-                    "Normal: ", pricing["normal"], "kr", "<br>",
+                    "Fullpris: ", pricing["normal"], "kr", "<br>",
```

Comment:
If we try to remove the use of `normal` should we rename the field in the config as well?

Reply:
Nah, that's fine. It's just when presenting the name to the end user I think it matters. Also that requires much more code, right?